tests/regression/mmpose.yml

```diff
@@ -58,6 +63,14 @@ ncnn:
     convert_image: *convert_image
     backend_test: False
     deploy_config: configs/mmpose/pose-detection_ncnn_static-256x192.py
+  pipeline_ncnn_static_fp32_256x256: &pipeline_ncnn_static_fp32_256x256
+    convert_image: *convert_image
+    backend_test: False
+    deploy_config: configs/mmpose/pose-detection_ncnn_static-256x256.py
+  pipeline_ncnn_static_int8_256x256: &pipeline_ncnn_static_int8_256x256
+    convert_image: *convert_image
+    backend_test: False
+    deploy_config: configs/mmpose/pose-detection_ncnn-int8_static-256x256.py
```

Comment:
Could ignore ncnn int8 test since it takes too much time.

Reply:
Alright👌